build.gradle.kts

```diff
@@ -19,28 +19,22 @@ gradlePlugin {
         create(pluginId) {
             id = pluginId
             implementationClass = "com.lovelysystems.gradle.LovelyGradlePlugin"
+            displayName = "Lovely Systems Project Helpers"
+            description = "Gradle Plugin for Lovely Systems Projects"
         }
     }
 }
 
 dependencies {
     testImplementation(gradleTestKit())
-    testImplementation("junit:junit:4.12")
     testImplementation(kotlin("test"))
     testImplementation(kotlin("test-junit"))
-    testImplementation("org.amshove.kluent:kluent:1.68")
+    testImplementation("org.amshove.kluent:kluent:1.72")
 }
 
 pluginBundle {
```

Comment:
preferred way seems to be to use `gradlePlugin` now for configuring this: https://docs.gradle.org/current/userguide/publishing_gradle_plugins.html#configure_the_plugin_publishing_plugin

Reply:
ah ok didn't know that, I am still new to gradle ;) yes then I think we can keep it as it is